js/services/http.js

```diff
@@ -3,7 +3,7 @@ define(function(require, exports) {
   const config = require('appConfig');
   const sharedState = require('atlas-state');
   const { Api:OHDSIApi, STATUS } = require('ohdsi-api');
-  const JSON_RESPONSE_TYPE = 'application/json';
```

Comment:
This looks like it replaced JSON_RESPONSE_TYPE with FHIR_JSON_RESPONSE_TYPE, and I don't think you should remove JSON_RESPONSE_TYPE...can you confirm?